packages/ses/src/transforms.js

```diff
@@ -32,13 +32,13 @@ function getLineNumber(src, pattern) {
 
 const htmlCommentPattern = new RegExp(`(?:${'<'}!--|--${'>'})`);
 
-export function rejectHtmlComments(src) {
+export function rejectHtmlComments(src, name) {
   const linenum = getLineNumber(src, htmlCommentPattern);
   if (linenum < 0) {
     return src;
   }
   throw new SyntaxError(
-    `possible html comment syntax rejected around line ${linenum}`,
+    `possible html comment syntax rejected around line ${linenum} of ${name}`,
```

Comment:
If you're changing these messages, let's clean them up!  The conventional `${name}:${linenum}` enables a Cmd-click from VSCode for example.

```suggestion
    `possible HTML comment syntax rejected around ${name}:${linenum}`,
```